rules/do-you-use-ai-for-ui/rule.md

```diff
@@ -0,0 +1,45 @@
+---
+type: rule
+archivedreason: 
+title: Do You Use AI in UI Design?
```

Comment:
Hye @lukecookssw 

- [ ] Please remove Capitalization from title (for consistency)

Reply:
Done